src/browser/browser.js

```diff
@@ -101,12 +102,18 @@ module.exports = class Browser {
     }
 
     _startCollectingCustomCommands() {
-        this._session.overwriteCommand("addCommand", (origCommand, name, ...rest) => {
-            if (!this._session[name]) {
```

Comment:
There was a mistake here. It is not possible to check the presence of a command on an element in this way. also, if the user has a custom command with the same name on the browser and the element (our case with `assertView` for example), then it was added only for the browser instance.

So I rewrite it to use command list for browser and element from history.